File: app/src/main/java/com/liusbl/listbinderssample/multi/PersonBinder.java
package com.liusbl.listbinderssample.multi;

import android.widget.TextView;

import com.liusbl.listbinders.BinderViewHolder;
import com.liusbl.listbinders.LayoutBinder;
import com.liusbl.listbinderssample.R;

class PersonBinder extends LayoutBinder<PersonListItem> {
    protected PersonBinder() {
        super(R.layout.item_multi_person, PersonListType.PERSON);
    }

    @Override
    public void onCreate(BinderViewHolder<PersonListItem> viewHolder) {
        // Empty
    }

    @Override
    public void onBind(BinderViewHolder<PersonListItem> viewHolder, PersonListItem item) {
        TextView nameTextView = viewHolder.itemView.findViewById(R.id.nameTextView);
        nameTextView.setAlpha(0);
        nameTextView.animate()
                .setDuration(500)
                .alpha(1)
                .start();
        nameTextView.setText(item.getName());
    }
}

File: app/src/main/java/com/liusbl/listbinderssample/multi/PersonBinder.kt
package com.liusbl.listbinderssample.multi

import com.liusbl.listbinders.BinderViewHolder
import com.liusbl.listbinders.LayoutBinder
import com.liusbl.listbinderssample.R
import com.liusbl.listbinderssample.multi.PersonListItem.ListType
import com.liusbl.listbinderssample.multi.PersonListItem.Person
import kotlinx.android.synthetic.main.item_multi_person.*

class PersonBinder : LayoutBinder<Person>(
    R.layout.item_multi_person,
    ListType.PERSON
) {
    override fun onBind(
        viewHolder: BinderViewHolder<Person>,
        item: Person
    ) {
        viewHolder.nameTextView.alpha = 0f
        viewHolder.nameTextView.animate()
            .apply { duration = 500 }
            .alpha(1f)
            .start()
        viewHolder.nameTextView.text = item.name
    }
}